include/utils.php

```diff
@@ -5523,17 +5523,6 @@ function suite_strrpos($haystack, $needle, $offset = 0, $encoding = DEFAULT_UTIL
     return strrpos($haystack, $needle, $offset);
 }
 
-/**
- * @param string $id
- * @return bool
- * @todo add to a separated common validator class
- */
-function isValidId($id)
```

Comment:
also called at modules/Users/ChangePassword.php:56

Reply:
Pushed to update